src/runtime_src/core/edge/skd/sk_daemon.cpp

```diff
@@ -67,12 +67,8 @@ static void sigLog(const int sig)
 	while (std::getline (backtrace, bt_line, '\n') )
 	  xrt_core::message::send(severity_level::error, "SKD", bt_line );
 #endif
-	struct sigaction act = {};
-	act.sa_handler = SIG_DFL;
-	sigemptyset(&act.sa_mask);
-	act.sa_flags = 0;
-	sigaction(sig, &act, nullptr);
-	raise(sig);
+	skd_inst->report_crash();
+	xrt_core::message::send(severity_level::error, "SKD", "SKD Reported crash!");
 	exit(sig);
```

Comment:
On Linux typically exit code in case of signals is 128+N where N is signal number. See

https://linuxconfig.org/list-of-exit-codes-on-linux

Please update the exit(sig) to something like exit(sig + 128)